openpype/hosts/houdini/plugins/create/create_vbd_cache.py

```diff
@@ -33,7 +33,18 @@ def create(self, subset_name, instance_data, pre_create_data):
         }
 
         if self.selected_nodes:
-            parms["soppath"] = self.selected_nodes[0].path()
+            sop_path = self.get_sop_node_path(self.selected_nodes[0])
+
+            if sop_path:
+                parms["soppath"] = sop_path
+            else:
+                self.log.debug(
+                    "Selection isn't valid. 'SOP Path' in ROP will be empty."
+                )
+        else:
+            self.log.debug(
+                "No Selection. 'SOP Path' in ROP will be empty."
+            )
```

Comment:
Note that this will ALSO be logged when the user had `Use selection` disabled on the Creator. Right? Which might seem a tad confusing?